LL/dal_corpri_arad.ttl

```diff
@@ -826,17 +831,429 @@
+    rel:descendantOf <#Labrada-747b0c60>;
+    rdfs:comment "h-Úi Chiru".
+
+<#Dicolla>
+    a foaf:Person;
+    irishRel:nomName "Dicuil";
+    irishRel:genName "Dicolla";
+    rel:descendantOf <#Labrada-747b0c60>;
+    rdfs:comment "h-Úi Dicolla".
+
+<#Garbine>
+    a foaf:Person;
+    irishRel:nomName "Garbene";
+    irishRel:genName "Garbine";
+    rel:descendantOf <#Labrada-747b0c60>;
+    rdfs:comment "h-Úi Garbine".
+
+<#Gerain>
+    a foaf:Person;
+    irishRel:nomName "Geran";
+    irishRel:genName "Gerain";
+    rel:descendantOf <#Labrada-747b0c60>;
+    rdfs:comment "h-Úi Gerain".
+
+<#Dáire-319eaab0>
+    a foaf:Person;
+    irishRel:nomName "Dáire";
+    irishRel:genName "Dáire";
+    rel:descendantOf <#Labrada-747b0c60>;
+    rdfs:comment "h-Úi Dáire".
+
+<#Iarnain-319eaab1>
+    a foaf:Person;
+    irishRel:nomName "Iarnan";
+    irishRel:genName "Iarnain";
+    rel:descendantOf <#Labrada-747b0c60>;
+    rdfs:comment "h-Úi Iarnain".
+
+<#Fidbadaig>
+    a foaf:Person;
+    irishRel:nomName "Fidbadach";
+    irishRel:genName "Fidbadaig";
+    rel:descendantOf <#Labrada-747b0c60>;
+    rdfs:comment "h-Úi Fidbadaig".
+
+<#Forannain>
+    a foaf:Person;
+    irishRel:nomName "Forannan";
+    irishRel:genName "Forannain";
+    rel:descendantOf <#Labrada-747b0c60>;
+    rdfs:comment "h-Úi Forannain".
+
+<#Maelglais>
+    a foaf:Person;
+    irishRel:nomName "Maelglas";
+    irishRel:genName "Maelglais";
+    rel:descendantOf <#Labrada-747b0c60>;
+    rdfs:comment "h-Úi Maelglais".
+
+<#DuibRatha>
+    a foaf:Person;
+    irishRel:nomName "Dub Ratha";
+    irishRel:genName "Duib Ratha";
+    rel:descendantOf <#Labrada-747b0c60>;
+    rdfs:comment "h-Úi Duib Ratha".
+
+<#Ingaind>
+    a foaf:Person;
+    irishRel:nomName "Ingand";
+    irishRel:genName "Ingaind";
+    rel:descendantOf <#Labrada-747b0c60>;
+    rdfs:comment "h-Úi Ingaind".
+
+<#Chellachain>
+    a foaf:Person;
+    irishRel:nomName "Cellachan";
+    irishRel:genName "Chellachain";
+    rel:descendantOf <#Labrada-747b0c60>;
+    rdfs:comment "h-Úi Chellachain".
+
+<#Bruggais>
+    a foaf:Person;
+    irishRel:nomName "Bruggas";
+    irishRel:genName "Bruggais";
+    rel:descendantOf <#Labrada-747b0c60>;
+    rdfs:comment "h-Úi Bruggais".
+
+<#Labrada-747b0c60>
+    owl:sameAs <#Labrada-88b47560>.
+
+<#Cormaic-319eaab2>
+    a foaf:Person;
+    irishRel:nomName "Cormac";
+    irishRel:genName "Cormaic";
+    rdfs:comment "IT é ranna h-Úa Cormaic la Laignib .i. an-gebthi h-Úi Gabla Fhini uile. & Cuthraige & h-Úi Gabla Roirend [...] Otá Culching co Dubathu et ó Glais Chriche i Cluanaib Cuada fri Laigis adiu. co Ath Lethnocht ic Sleibti. co téit i nUssin. fri Uib Bairrchi atuaid & angebthi .h. Trena & h. Chuirc. is la Cormac sin uile.";
+    owl:sameAs <#Cormac>.
+
+<#EithniGabulfhota>
+    a foaf:Person;
+    foaf:gender "female";
+    irishRel:nomName "Eithni Gabulfhota";
+    rel:childOf <#Cormaic-319eaab2>;
+    rel:spouseOf [
+      a foaf:Person;
```

Comment:
Something weird is going on. I can't find <#EithniGabufhota> on my branch (searching via Atom) but she's there if I look for her on the online editor. And, on the online editor, she is foaf:gender "female". Any ideas what this is all about?